### PR TITLE
Add intro for primary_file_spec

### PR DESCRIPTION
**Pull request by rfrenchseti**
- Were any Django, import pipeline, table_schema, or dictionary files modified? Y
  - Database used: opus3_test_200304
  - All Django tests pass: Y
- Were any JavaScript or CSS files modified? N
  - JSHINT run on all affected files: NA
  - Tested on Chrome, Firefox, Safari, and iOS: N
    (Remember to test all browser sizes)
  - Tested for race conditions (with delayed API calls if applicable): NA

Description of changes:

Added intro paragraph to the primary file spec search widget explaining how to search without using file extensions.

Known problems:

None.

### CODE REVIEW

**Comment by matthewtiscareno** [comment]
Hi Rob.  Can I see how this change looks in OPUS?

**Comment by matthewtiscareno** [comment]
SpecX ==> Spec
has different ==> uses different
each mission and instrument ==> different data product types,
eliminate ==> do not include
from ==> in
and use ==> and that you use

File: opus/import/table_schemas/obs_pds.json [hunk 1]
```diff
@@ -177,7 +177,7 @@
         "pi_display": 1,
         "pi_display_results": 1,
         "pi_form_type": "STRING",
-        "pi_intro": null,
+        "pi_intro": "The Primary File SpecX has different filename extensions for each mission and instrument and in many cases specifies the label file instead of the data file. Thus we suggest that you eliminate the extension from your search and use the \"contains\" query type to guarantee the best match.",
```
Comment (matthewtiscareno) [inline opus/import/table_schemas/obs_pds.json]: SpecX ==> Spec
has different ==> uses different
each mission and instrument ==> different data product types,
eliminate ==> do not include
from ==> in
and use ==> and that you use

**Review by matthewtiscareno — CHANGES_REQUESTED** [review]
Good work, Rob!  I have a few copy editing suggestions.

File: opus/import/table_schemas/obs_pds.json (hunk 1 above)
Comment (rfrenchseti) [reply opus/import/table_schemas/obs_pds.json]: Made all changes except different data product types, which is now "different filename extensions depending on the mission and instrument, and in many cases"

**Review by matthewtiscareno — APPROVED** [review]
Perfect.  Thanks.

**Pull request merged**